test/end-to-end/specs/imageBuilding.test.js

```diff
@@ -119,6 +122,7 @@ describe("View Blueprint Page", function() {
 
   describe("Delete Image Page", function() {
     before(function() {
+      viewBlueprintPage.imagesTab.click();
```

Comment:
@henrywang I'm not sure whether this line was the right fix for this test or not, but now the `Delete Image Page` tests are passing with this update. 

As part of the PF4 migration, I moved the Download action to the actions menu. This required me to add `viewBlueprintPage.imageMoreButton.click();` above (line 63) so that the Download action could be found. But I think this also caused the same call on the next line (125) to result in dismissing the menu, making the Delete action not visible. Based on this assumption, I added `viewBlueprintPage.imagesTab.click();` here so that if the menu is visible, it is dismissed, so that when the `imageMoreButton` is clicked, it will open the menu again.